masterlist.yaml

```diff
@@ -952,6 +952,16 @@ plugins:
       - crc: 0x23E4EB08
         util: 'TES5Edit v4.0.4b'
 
+  - name: 'dD - Realistic Ragdoll Force - (Realistic|Reduced|Medium|High)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrim/mods/601/'
+        name: 'Realistic Ragdolls and Force'
+    after: [ 'Requiem.esp' ]
+    msg:
+      - <<: *alreadyInX
+        subs: [ '**XPMSE.esp**' ]
+        condition: 'active("XPMSE.esp")'
```

Comment:
IIRC some of the Ragsoll plugins can be used with XPMSE, the game settings are slightly different.

Reply:
It was the same in the SE masterlist at one point, possibly copied over.